web/src/enterprise/campaigns/detail/changesets/ChangesetNode.tsx

```diff
@@ -114,6 +115,13 @@ export const ChangesetNode: React.FunctionComponent<ChangesetNodeProps> = ({
                             >
                                 {node.title}
                             </LinkOrSpan>
+                            {node.labels.length > 0 && (
+                                <span className="ml-2">
+                                    {node.labels.map((label, i) => (
+                                        <ChangesetLabel label={label} key={i} />
```

Comment:
This would be a better key since labels are unique
```suggestion
                                        <ChangesetLabel label={label} key={label.name} />
```

Reply:
I did not find guarantees in the docs they would be 🤔

Reply:
I mean, it's the only sensible interpretation, no? When trying to create a new label:

![image](https://user-images.githubusercontent.com/10532611/73480001-50a0e380-4399-11ea-951c-011a95c7c277.png)

Reply:
hmm probably it is unique. Would have to revisit for phabricator, gitlab in the future, but changing for now